examples/authorization/jupyter_temporary_config.py

```diff
@@ -0,0 +1,14 @@
+from jupyter_server.services.auth.manager import AuthorizationManager
+
+
+class TemporaryServerPersonality(AuthorizationManager):
+    """Manager class that makes Jupyter Server a read-only server."""
+
+    def is_authorized(self, handler, subject, action, resource):
+        """Only allows `read` operations."""
+        if action == 'write' and resource == 'contents':
+            return False
+        return True
+
+
+c.ServerApp.authorization_manager_class = TemporaryServerPersonality
```

Comment:
"manager" pattern sounds inline with the current terminology ('kernel_manager"....), but if we want to go out at some point from that, 'authorizer" may be another option.